go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/gardener/dependency-watchdog v0.7.0
 	github.com/gardener/etcd-druid v0.15.3
 	github.com/gardener/hvpa-controller/api v0.5.0
-	github.com/gardener/machine-controller-manager v0.45.0
+	github.com/gardener/machine-controller-manager v0.48.1
```

Comment:
Aren't there any changes to the MCM CRDs (currently maintained under `extensions/pkg/controller/worker/tempaltes/crds`). 

I think we should also put comments on these MCM CRDs from which source they are maintained. For example https://github.com/gardener/gardener/blob/a243225097a891aa497f70bdf125424bd5cd55d3/pkg/operation/botanist/component/etcd/crds/templates/crd-druid.gardener.cloud_etcdcopybackupstasks.yaml#L1, https://github.com/gardener/gardener/blob/9b4bee5e9999a486f99e313c903912c4bc8d99d7/pkg/operation/botanist/component/vpa/templates/crd-verticalpodautoscalercheckpoints.tpl.yaml#L2. 
Then you know which version of the CRDs you use.